### PR TITLE
[MSGraph] add live test for MSGraph

### PR DESCRIPTION
**Pull request by VeryEarly**
## Checklist

- [ ] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** select appropriate branch. Cmdlets from Autorest.PowerShell should go to [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch. 
* **SHOULD** make the title of PR clear and informative, and in the present imperative tense. 
* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. Add changelog in description section if PR goes into [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** have approved design review for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) ([_Microsoft internal only_](../blob/main/CONTRIBUTING.md#onboarding)) with following situations
    * Create new module from scratch
    * Create new resource types which are not easy to conform to [Azure PowerShell Design Guidelines](../blob/main/documentation/development-docs/design-guidelines)
    * Create new resource type which name doesn't use module name as prefix
    * Have design question before implementation
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** introduce [breaking changes](../blob/main/documentation/breaking-changes/breaking-changes-definition.md) in Az minor release except preview version.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

File: src/Resources/LiveTests/msgraphtest2.cer [hunk 1]
```diff
@@ -0,0 +1,24 @@
+-----BEGIN CERTIFICATE-----
```
Comment (vidai-msft) [inline src/Resources/LiveTests/msgraphtest2.cer:1]: Does it have security concern when exposing the certificate? Is it possible to generate it by the command New-SelfSignedCertificate?

File: src/Resources/LiveTests/TestLiveScenarios.ps1 [hunk 2]
```diff
@@ -0,0 +1,113 @@
+Invoke-LiveTestScenario -Name "Test Application" -Description "Test the process of create an application." -NoResourceGroup -PowerShellVersion Latest -ScenarioScript `
```
Comment (vidai-msft) [inline src/Resources/LiveTests/TestLiveScenarios.ps1:1]: The -PowerShellVersion parameter here looks like not necessary.

File: src/Resources/LiveTests/TestLiveScenarios.ps1 [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+        if ($app1) {
+            Remove-AzADApplication -DisplayName $appName1
+        }
+        if ($app2) {
+            Remove-AzADApplication -DisplayName $spName2
+        }
+    }
+}
+
+Invoke-LiveTestScenario -Name "Test Group Member" -Description "Test the process of create groups and members." -NoResourceGroup -PowerShellVersion Latest -ScenarioScript `
```
Comment (vidai-msft) [inline src/Resources/LiveTests/TestLiveScenarios.ps1:63]: The -PowerShellVersion parameter here looks like not necessary.

File: src/Resources/LiveTests/TestLiveScenarios.ps1 [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+    try {
+        $today = (Get-Date).tostring('yyyy-MM-dd')
+        $groupName1 = $today + 'testgroup' + (New-LiveTestRandomName)
+        $groupName2 = $today + 'testgroup' + (New-LiveTestRandomName)
+        $userName = $today + 'testuser' + (New-LiveTestRandomName)
+        
+        $groupMailNickName1 = New-LiveTestRandomName
+        $groupMailNickName2 = New-LiveTestRandomName
+        $userMailNickName = New-LiveTestRandomName
+        $userPrincipalName = $userMailNickName + 'microsoft.com#EXT#@AzureSDKTeam.onmicrosoft.com'
```
Comment (vidai-msft) [inline src/Resources/LiveTests/TestLiveScenarios.ps1:74]: Is it possible to hide the internal name?

**Review by vidai-msft — CHANGES_REQUESTED** [review]

File: src/Resources/LiveTests/msgraphtest2.cer (hunk 1 above)
Comment (VeryEarly) [reply src/Resources/LiveTests/msgraphtest2.cer:1]: this is a random certificate existed in our repo for quite a while:
https://github.com/Azure/azure-powershell/blob/main/src/Resources/MSGraph.Autorest/test/msgraphtest2.cer

File: src/Resources/LiveTests/TestLiveScenarios.ps1 (hunk 4 above)
Comment (VeryEarly) [reply src/Resources/LiveTests/TestLiveScenarios.ps1:74]: this domain name has to be correct otherwise user cannot be created.

**Review by vidai-msft — APPROVED** [review]

**Pull request merged**